package.json

```diff
@@ -9,7 +9,7 @@
     "d3-tip":"~0.9.1",
     "jquery": "~3.3.1",
     "underscore":"",
-    "bootstrap": "~4.1.3",
+    "bootstrap": "~4.3.1",
```

Comment:
It looks like popper.js also has a new version of 1.14.7. Though I guess the ^ syntax will pick this up.

https://stackoverflow.com/a/22345808/3708872

So that's probably ok.

Reply:
Though we might want to avoid using the ^ syntax for slightly more controlled versions, then it will only pick up the latest minors.